cocotb/share/lib/vhpi/VhpiCbHdl.cpp

```diff
@@ -491,7 +491,7 @@ int VhpiLogicSignalObjHdl::set_signal_value(long value, gpi_set_action_t action)
         case vhpiLogicVecVal: {
             int i;
             for (i=0; i<m_num_elems; i++)
-                m_value.value.enumvs[m_num_elems-i-1] = value&(1L<<i) ? vhpi1 : vhpi0;
+                m_value.value.enumvs[m_num_elems-i-1] = value&(1LL<<i) ? vhpi1 : vhpi0;
```

Comment:
Strictly speaking this should be
```suggestion
                m_value.value.enumvs[m_num_elems-i-1] = value&(UINT64_C(1)<<i) ? vhpi1 : vhpi0;
```
But it doesn't matter anyway.

Reply:
Thanks, I didn't know about those macros